src/RTCRtpSender.ts

```diff
@@ -1,50 +1,60 @@
-import { NativeModules } from 'react-native';
-
-import MediaStreamTrack from './MediaStreamTrack';
-import RTCRtpCapabilities, { senderCapabilities, DEFAULT_AUDIO_CAPABILITIES } from './RTCRtpCapabilities';
-import RTCRtpSendParameters, { RTCRtpSendParametersInit } from './RTCRtpSendParameters';
+import { NativeModules } from "react-native";
+import RTCDTMFSender from "./RTCDTMFSender";
+
+import MediaStreamTrack from "./MediaStreamTrack";
+import RTCRtpCapabilities, {
+    senderCapabilities,
+    DEFAULT_AUDIO_CAPABILITIES,
+} from "./RTCRtpCapabilities";
+import RTCRtpSendParameters, {
+    RTCRtpSendParametersInit,
+} from "./RTCRtpSendParameters";
 
 const { WebRTCModule } = NativeModules;
 
-
 export default class RTCRtpSender {
     _id: string;
     _track: MediaStreamTrack | null = null;
     _peerConnectionId: number;
     _rtpParameters: RTCRtpSendParameters;
+    _dtmf: RTCDTMFSender;
 
     constructor(info: {
-        peerConnectionId: number,
-        id: string,
-        track?: MediaStreamTrack,
-        rtpParameters: RTCRtpSendParametersInit
+        peerConnectionId: number;
```

Comment:
Same issue here, no longer functional.

Reply:
😕 It was working on my end, apart from formatting, I have added canInsertDtmf check before sending the dtmf on native side, can you please provide the logs

Reply:
also can you please confirm as provided [here](https://www.w3.org/TR/webrtc/#caninsertdtmf-algorithm) all the points are valid in your case

Reply:
Just realised, nope that is my bad, as that is a type definition you can indeed use `;` instead of `,`
Results of looking over code quick 😆 can mark them as resolved 👍🏻

Reply:
Oo, no problem thanks